docs/cookbook/installing-on-mac-os-and-windows.md

```diff
@@ -85,12 +85,8 @@ To install Composer dependencies, from the folder into which you downloaded eZ P
 composer install
 ```
 
-After a moment the installer will ask you to provide a few parameters:
-
-1. Choose a [secret](http://symfony.com/doc/current/reference/configuration/framework.html#secret); it should be a random string, made up of up to 32 characters, numbers, and symbols. This is used by Symfony when generating [CSRF tokens](https://symfony.com/doc/current/security/csrf.html), [encrypting cookies](http://symfony.com/doc/current/cookbook/security/remember_me.html), and for creating signed URIs when using [ESI (Edge Side Includes)](https://symfony.com/doc/current/http_cache/esi.html).
-1. You can accept the default options for `database_driver`, `database_host` and `database_port`.
-1. Select a `database_name` or accept the default one.
-1. Provide your `database_user` and `database_password`.
+At this point you  can change any installation parameters, such as `DATABASE_NAME` or `DATABASE_USER`,
+in the `.env` file.
```

Comment:
There are other database related variables:
- `DATABASE_PASSWORD`
- `DATABASE_HOST`
- `DATABASE_PORT`

What's more, database can be configured via `DATABASE_URL` env `DATABASE_URL=mysql://user:password@host:port/name` which is now recommended by Symfony. We still use separate variables due to our docker configuration.

I wouldn't remove information about secret string - it's very important and it's still there. Integrators are meant to change in `APP_SECRET` variable.